whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmVersionInfoRepository.java

```diff
@@ -2,7 +2,6 @@
 
 import org.slf4j.Logger;
```

Comment:
Change the java doc from createInitialSnapshot, you swap the same of the table from version_information to version

Reply:
done